dss/operations/checkout.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+
+
+logger = logging.getLogger(__name__)
+
+checkout = dispatch.target("checkout", help=__doc__)
+
+
+def verify_delete(handler, bucket, key):
```

Comment:
I think if this was:
```
    logger.warning(f'attempting removal of key: {key}')
    try:
        handler.delete(bucket, key)
        handler.get(bucket=bucket, key=key)
    except BlobNotFoundError:
        logger.warning(f'Success! unable to locate key {key} in bucket: {bucket} ')
```
Then you wouldn't need the `try/except` elsewhere.

Reply:
+1